parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -280,6 +349,13 @@ Update-CNIConfig(`$podCIDR, `$masterSubnetGW)
 
 try
 {
+    
+    if (`$global:NetworkPolicy -eq "azure") {

```

Comment:
Since with azure cni, kubelet start is much simpler, it seems to me we can write a much simpler kubeStartStr in case of azure network policy instead of return here without using rest of the cmdlet.